README.md

```diff
@@ -103,6 +104,18 @@ additional metrics!
 
 See the [`Documentation`](Documentation) directory for documentation of the exposed metrics.
 
+### Metrics Deprecation
+* **Pod none generic resource metrics below are marked as deprecated and will be finally removed in kube-state-metrics v2.0.0.**
```

Comment:
nit: this reads awkwardly.

"The following non-generic resource metrics for pods are marked deprecated.  They will be removed in kube-state-metrics v2.0.0"

Reply:
Thanks for pointing this out. English writing skill should be enhanced. :-)